cvmfs/publish/cmd_abort.cc

```diff
@@ -38,114 +48,111 @@ int CmdAbort::Main(const Options &options) {
     throw;
   }
 
-  if (settings->transaction().in_enter_session()) {
-    throw EPublish(
-      "aborting a transaction is unsupported within the ephemeral "
-      "writable shell",
-      EPublish::kFailInvocation);
-  }
-
-  if (!SwitchCredentials(settings->owner_uid(), settings->owner_gid(),
-                         false /* temporarily */))
-  {
-    throw EPublish("No write permission to repository",
-                   EPublish::kFailPermission);
-  }
-
-  if (HasPrefix(GetCurrentWorkingDirectory() + "/",
-                settings->transaction().spool_area().union_mnt() + "/",
-                false /* ignore_case */))
-  {
-    LogCvmfs(kLogCvmfs, kLogStdout,
-             "Current working directory is in %s.  Please release, "
-             "e.g. by 'cd $HOME'.",
-             settings->transaction().spool_area().union_mnt().c_str());
-    return 1;
-  }
-
-  if (!options.Has("force")) {
-    LogCvmfs(kLogCvmfs, kLogStdout | kLogNoLinebreak,
-             "You are about to DISCARD ALL CHANGES OF THE CURRENT TRANSACTION "
-             "for %s!  Are you sure (y/N)? ", settings->fqrn().c_str());
-    char answer[] = {0, 0, 0};
-    char *rv_charp = fgets(answer, 3, stdin);
-    if (rv_charp && (answer[0] != 'Y') && (answer[0] != 'y'))
-      return EINTR;
-  }
+    if (!SwitchCredentials(settings->owner_uid(), settings->owner_gid(),
```

Comment:
It seems that in the following lines some automatic style changes were applied. That often makes the `diff`s harder to read.